Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -37,8 +37,19 @@ public class VobCreator : SingletonBehaviour<VobCreator>
 
         private Dictionary<PxVobType, GameObject> parentGosTeleport = new();
         private Dictionary<PxVobType, GameObject> parentGosNonTeleport = new();
-        private PxVobType[] nonTeleportTypes = { PxVobType.PxVob_oCItem , PxVobType.PxVob_oCMobLadder };
-        
+
+        private PxVobType[] nonTeleportTypes =
+        {
+            PxVobType.PxVob_oCItem,
+            PxVobType.PxVob_oCMobLadder,
+            PxVobType.PxVob_zCTrigger,
+            PxVobType.PxVob_zCTriggerUntouch,
+            PxVobType.PxVob_oCTriggerScript,
+            PxVobType.PxVob_zCTriggerList,
+            PxVobType.PxVob_zCTriggerWorldStart,
+            PxVobType.PxVob_oCTriggerChangeLevel
```

Comment:
Thanks for adding these. Nevertheless, in the end, they won't have meshes, right? So mostly not needed.
But we can have them here, sure.